specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-07-26-preview/examples/ledger/CreateUserDefinedEndpoint.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "api-version": "2024-07-26-preview",
```

Comment:
How did we decide on this Api version?

Reply:
In the control plane it's based on PR creation date, I will update the same in this PR